src/music/dmusic.cpp

```diff
@@ -572,27 +574,15 @@ static void TransmitNotesOff(IDirectMusicBuffer *buffer, REFERENCE_TIME block_ti
 		TransmitChannelMsg(_buffer, block_time + 10, MIDIST_CONTROLLER | ch, MIDICT_SUSTAINSW, 0);
 		TransmitChannelMsg(_buffer, block_time + 10, MIDIST_CONTROLLER | ch, MIDICT_MODE_RESETALLCTRL, 0);
 	}
+
+	TransmitStandardSysex(_buffer, block_time + 20, MidiSysexMessage::ResetGM);
+	TransmitStandardSysex(_buffer, block_time + 30, MidiSysexMessage::RolandSetReverb);
```

Comment:
Either these lines need to be in the spot where the old pitch bend reset was or the comment that follows is outdated, as there aren't any additional control messages.